### DISCUSSION TITLE
Does it support multiple collectors

### DISCUSSION THREAD

**Discussion by mitht** [apache/skywalking#12279] [Q&A]
How can the SkyWalking Java Agent send trace data to both the SkyWalking OAP Server and the OpenTelemetry Collector simultaneously?
Is it possible to configure two collector addresses at the same time?
SW_AGENT_COLLECTOR_BACKEND_SERVICES=oap-server:11800
SW_AGENT_COLLECTOR_BACKEND_SERVICES=otel-agent.open-telemetry:11800

**Comment by wu-sheng** [comment]
You can't. First, otel doesn't have skywalking trace receiver support, at least isn't from us. Then, agent doesn't rely on two backends. 

You could use backend trace exporter.
